spec/models/organization_capacity_spec.rb

```diff
@@ -19,6 +19,7 @@
         before do
           create :client_with_status, status: status[0], vita_partner: organization
         end
+
```

Comment:
@bytheway875 to avoid this new status from getting included in the `active_client_count`, can I just add `AND tax_returns.status != 130` to the existing SQL views (organization_capacities_v01.sql) for selecting client_ids or should I generate a new view?

Reply:
@bytheway875 we ended up generating version 3 of the organization_capacity view and added the change and it seemed to do the trick!

Reply:
yes! That's right -- you've got to add a new version of the view anytime you make a change. The sql gets added directly to schema.rb to execute, and we use the versions to make sure that we're not editing schema.rb directly. @embarnard